src/components/Project/AragonClient.js

```diff
@@ -97,14 +99,14 @@ class AragonClient extends React.Component {
               <Entry>
                 <Module>
                   <div className="badge-container">
-                    <StyledBadge className="brown">App center</StyledBadge>
+                    <StyledBadge className="brown">App Center</StyledBadge>
                     <AragonOneBadge>Aragon One</AragonOneBadge>
                   </div>
-                  <h3>Upgrading apps from the UI</h3>
+                  <h3>App Center: App upgrades</h3>
```

Comment:
> App upgrades

Does "app" need to be capitalized here? I notice you don't capitalize after the colon anywhere else.